backend/src/test/java/ai/verta/modeldb/FindProjectEntitiesTest.java

```diff
@@ -2580,4 +2580,71 @@ public void findExperimentRunsByExperimentTest() {
 
     LOGGER.info("FindExperimentRuns by experiment test stop.........");
   }
+
+  @Test
+  public void findExperimentRunsByExperimentNameTest() {
+    LOGGER.info("FindExperimentRuns by experiment name test start.........");
+
+    FindExperimentRuns findExperimentRuns =
+        FindExperimentRuns.newBuilder()
+            .setProjectId(project1.getId())
+            .addPredicates(
+                KeyValueQuery.newBuilder()
+                    .setKey("experiment.name")
+                    .setValue(Value.newBuilder().setStringValue(experiment1.getName()).build())
+                    .build())
+            .build();
+
+    FindExperimentRuns.Response response =
+        experimentRunServiceStub.findExperimentRuns(findExperimentRuns);
+    LOGGER.info("FindExperimentRuns Response : " + response.getExperimentRunsCount());
+    assertEquals(
+        "ExperimentRun count not match with expected experimentRun count",
+        2,
+        response.getExperimentRunsCount());
+    assertEquals(
+        "ExperimentRun not match with expected experimentRun",
+        experiment1.getId(),
+        response.getExperimentRunsList().get(0).getExperimentId());
```

Comment:
Is the output sorted such that we can depend on ordering?  
Perhaps better to assert the list contains experiment1 id.